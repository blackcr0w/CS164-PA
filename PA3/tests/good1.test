/home/ff/cs164/etc/../assignments/PA3/tests/good1.test