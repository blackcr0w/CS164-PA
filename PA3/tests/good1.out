#name "tests/good1.cl"
#1 CLASS
#1 TYPEID Hello
#1 '{'
#2 '}'
#2 ';'
#4 CLASS
#4 TYPEID Goodbye
#4 '{'
#5 '}'
#5 ';'
